bedrock/mozorg/forms.py

```diff
@@ -223,6 +223,19 @@ class ContributeForm(forms.Form):
     office_fax = forms.CharField(widget=HoneyPotWidget, required=False)
 
 
+class ContributeTasksForm(forms.Form):
+    required_attr = {'required': 'required'}
+
+    email = forms.EmailField(widget=EmailInput(attrs=required_attr))
+    name = forms.CharField(widget=forms.TextInput(attrs=required_attr))
+    privacy = forms.BooleanField(widget=PrivacyWidget)
+    category = forms.CharField(widget=widgets.HiddenInput(attrs={'value': 'dontknow'}))
```

Comment:
Is `dontknow` the only valid value for this field? It seems so, but I don't see any validation of this. We should probably remove this field and send `dontknow` as the hard coded value in the view. Relying on this to come back unchanged from the browser won't work.